src/js/components/Signature.jsx

```diff
@@ -5,14 +5,20 @@ import { ordinalHour } from '../utils/date.js'
 import Alert from './Alert.jsx'
 import { VALIDATED_WITH_ERRORS, SIGNED } from '../constants/statusCodes.js'
 
-const showReceipt = (code, timestamp, receipt) => {
+const showReceipt = ({status, timestamp, receipt, filingPeriod, email}) => {
+  const code = status.code
   if (code !== SIGNED) return null
 
   return (
-    <Alert type="success" heading="HMDA data accepted">
+    <Alert type="success" heading="HMDA filing completed">
       <div>
-        <p>Your submission was received on <strong>{ordinalHour(new Date(timestamp))}</strong>.</p>
-        <p>Your receipt number is <strong>{receipt}</strong>.</p>
+        <hr/>
```

Comment:
I think we'd be ok without the `<hr>` tags in this alert, including the one below. We haven't used them in other alerts and there's not a real change in content here. The messaging is all about filing completion.